src/app/components/forms/user-details-form/user-details-form.component.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../styles/colors';
+
+.user-details-form-button-container {
+  margin: 40px 0;
+  text-align: center;
+
+  .user-details-form-save-message {
+    color: green;
+  }
+
+  .user-details-form-error-message {
+    color: $color-cerise-500;
+  }
+
+  .user-details-form-button {
+    display: block;
+    margin: 15px auto 30px;
```

Comment:
Properties should be ordered display, min-width, margin

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)